packages/components/src/internal/components/forms/QueryFormInputs.tsx

```diff
@@ -379,16 +382,29 @@ export class QueryFormInputs extends React.Component<QueryFormInputsProps, State
                             );
                         default:
                             return (
-                                <TextInput
-                                    key={i}
-                                    queryColumn={col}
-                                    value={value ? String(value) : value}
-                                    allowDisable={allowFieldDisable}
-                                    initiallyDisabled={shouldDisableField}
-                                    onToggleDisable={this.onToggleDisable}
-                                    addLabelAsterisk={showAsteriskSymbol}
-                                    renderFieldLabel={renderFieldLabel}
-                                />
+                                <>
+                                    <TextInput
```

Comment:
Does it make sense to have `TextInput` handle rendering of `InternalSpacesWarning` ? Also, if disabled, maybe the warning shouldn't be displayed.

Reply:
The layout was a bit off because `TextInput` doesn't know about the surrounding structure. It seemed better to leave that out here. 

I think I will leave the warning displayed even if disabled. It might be frustrating since the value wouldn't be able to be changed, but it will help in troubleshooting if it happens that there is some non-editable input with this malady.